Dockerfile_archive

```diff
@@ -1,4 +1,17 @@
-FROM docker.io/debian:testing-slim AS build
+# Pull (Pre-Built):
+#   $ docker pull ghcr.io/jelmer/janitor/archive:latest
```

Comment:
for consistency please don't use docker here, just podman/buildah

Reply:
Noted. Switched!